tools/phenomenal/w4m/mtbls-dwnld_config.xml

```diff
@@ -1,312 +0,0 @@
-
-		<data format="tabular" name="w4m_all_assays">
-			<discover_datasets pattern="__designation_and_ext__" directory="w4m" visible="true"/>
-		</data>
-
-		<collection name="mzML" type="list" label="mzML files">
-			<discover_datasets pattern="__name_and_ext__" directory="mzML" format="mzml"/>
-		</collection>
-
-		<collection name="mzData" type="list" label="mzData files">
-			<discover_datasets pattern="__name_and_ext__" directory="mzData" format="mzdata"/>
-		</collection>
-
-		<data name="w4m_variable_metadata" label="W4M variable metadata.tsv" format="tabular"/>
-        <data name="w4m_sample_metadata" label="W4M sample metadata.tsv" format="tabular"/>
-        <data name="w4m_data_matrix" label="W4M data matrix.tsv" format="tabular"/>
-
-        <!-- Using new type by inheriting only from HTML, no class definition -->
-   		<!--     <data name="isatab_output_inherit_html" label="ISA-Tab of $study - inherit HTML.isatabhtml" format="isatabhtml"/> -->
-        <!-- Put the following line inside "config/datatypes_conf.xml":
-	  <datatype extension="isatabhtml" type="galaxy.datatypes.text:Html" subclass="True"/>
--->
-
-		<!-- Using new composite type, creating new class. -->
-        <!-- <data name="isatab_output_composite" label="ISA-Tab of $study - composite.isatab" format="isatab"/> -->
-        <!-- Put the following line inside "config/datatypes_conf.xml":
-	  				<datatype extension="isatab" type="galaxy.datatypes.isatab:ISATab"/>
-			Then make a symbolic link to "isatab.py" inside "lib/galaxy/datatypes".
-		-->
-    </outputs>
-
-	<!-- Tests {{{1 -->
-	<tests>
-		<test>
-			<param name="study" value="MTBLS30"/>
-			<param name="assay" value="a_york_src_GCMS_metabolomics_metabolite profiling_mass spectrometry.txt"/>
-			<output name="html_output" file="mtbls30.html"/>
-			<output name="w4m_sample_metadata" file="mtbls30-gcms-w4m-sample-metadata.tsv"/>
-			<output name="w4m_variable_metadata" file="mtbls30-gcms-w4m-variable-metadata.tsv"/>
-			<output name="w4m_data_matrix" file="mtbls30-gcms-w4m-sample-variable-matrix.tsv"/>
-		</test>
-		<test>
-			<param name="study" value="MTBLS30"/>
-			<param name="assay" value="a_york_src_LCMS_metabolomics_metabolite profiling_mass spectrometry.txt"/>
-			<output name="html_output" file="mtbls30.html"/>
-			<output name="w4m_sample_metadata" file="mtbls30-lcms-w4m-sample-metadata.tsv"/>
-			<output name="w4m_variable_metadata" file="mtbls30-lcms-w4m-variable-metadata.tsv"/>
-			<output name="w4m_data_matrix" file="mtbls30-lcms-w4m-sample-variable-matrix.tsv"/>
-		</test>
-	</tests>
-
-	<!-- Help {{{1 -->
-    <help>
-<!-- @@@BEGIN_RST@@@ -->
-
-=======================
-Metabolights Downloader
-=======================
-
-A tool to download data from MetaboLights studies. You can download either a private or a public study, using `wget` as well as `aspera` methods.
-
-The following outputs are available:
- - A single assay in W4M three files format.
- - All assays in W4M three files format.
```

Comment:
three or tree?

Reply:
Are all outputs (W4M, mzData, mzML) generated automatically or you need to actively request a particular one? Duplication of datasets could be undesirable given large files. Or is this duplication handled somehow?